appcenter-post-build.sh

```diff
@@ -1,4 +1,41 @@
-# Script is to be included in the root directory of your repository. 
-curl -H "Content-Type: application/json" -H "authToken: 349ab29a-xtab-423b-a5hc-5623bc39b8c8" --data '{}' https://api.example.com/endpoint
+# Sample post-build script for Github PR checks
```

Comment:
Let's talk about how we want to clean this file up and make it a little bit more dynamic for end users. Forcing them to configure things we can discover seems like it deserves some improvement.

Reply:
Should leverage environment variables in the build configuration for owner/repo portion of the path and Github token. We can execute something like `SHA=git rev-parse --short HEAD 2> /dev/null | sed "s/\(.*\)/@\1/"` to get the current sha into an environment var.

Reply:
Got it, will investigate and let you know the results.